cli/templatecreate.go

```diff
@@ -166,9 +166,10 @@ func (r *RootCmd) templateCreate() *clibase.Cmd {
 			Value:       clibase.StringOf(&variablesFile),
 		},
 		{
-			Flag:        "variable",
-			Description: "Specify a set of values for Terraform-managed variables.",
-			Value:       clibase.StringArrayOf(&variables),
+			Flag:          "variable",
+			Description:   "Specify a set of values for Terraform-managed variables.",
+			Value:         clibase.StringArrayOf(&variables),
+			FlagShorthand: "V",
```

Comment:
`-V` is always confusing as it usually means `--version` or `--verbose`.

Maybe we should another alias, `-var`, instead? Alternatively, we can add support for [TF_VAR_](https://github.com/coder/coder/issues/8279).